cmd/crio/config.go

```diff
@@ -18,8 +23,29 @@ it later with **--config**. Global options will modify the output.`,
 			Name:  "default",
 			Usage: "Output the default configuration (without taking into account any configuration options).",
 		},
+		&cli.StringFlag{
+			Name:        "migrate-defaults",
+			Aliases:     []string{"m"},
+			Destination: &from,
+			Usage: `Migrate the default config from a certain version.
+    To run a config migration, just select the input config via the global
+    '--config,-c' command line argument, for example:
+
+    crio -c /etc/crio/crio.conf.d/00-default.conf config -m 1.17
+
+    The migration will print converted configuration options to stderr and will
+    output the resulting configuration to stdout. Please note that the migration
+    will only convert changed defaults between CRI-O versions by preserving
+    custom settings in the configuration.`,
```

Comment:
suggested edit:

`Please note that the migration will overwrite any fields that have changed defaults between versions. To save a custom configuration change, it should be in a drop-in configuration file instead.`

Reply:
Fixed :+1: